ledger/src/blockstore_processor.rs

```diff
@@ -582,6 +588,11 @@ fn process_entries_with_callback(
                 }
             }
             EntryType::Transactions(transactions) => {
+                block_min_prioritization_fee_cache
+                    .write()
```

Comment:
I wonder if we'll have much contention on this. If so, we can Arc-wrap each block in the cache and pass just the one &RwLock<PrioritizationFee> into blockstore_processor

Reply:
yeah agreed, don't want the replay threads all synchronizing on the lock

Reply:
Excellent point!